apps/legacy-api/src/pipes/NetworkValidationPipe.ts

```diff
@@ -5,14 +5,15 @@ import {
   PipeTransform
 } from '@nestjs/common'
 
-export type SupportedNetwork = 'mainnet' | 'testnet'
+export type SupportedNetwork = 'mainnet' | 'testnet' | 'regtest'
```

Comment:
I don't think there's a need for this 
```suggestion
export type SupportedNetwork = 'mainnet' | 'testnet'
```